Szkodzinski_Kacper/lab0.py

```diff
@@ -0,0 +1,76 @@
+def main():
+    print("Kacper Szkodzinski - sortowanie liczb.")
```

Comment:
takie informacje są widoczne w gicie, po to mam Wasze loginy. Nie wrzucaj ich do kodu.